src/DDoSX/Entities/DomainProperty.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace UKFast\SDK\DDoSX\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $name
+ * @property integer $value
```

Comment:
nit: 
```suggestion
 * @property int $value
```

we seem to use `int` elsewhere